sig-cloud-provider/annual-report-2021.md

```diff
@@ -23,62 +32,59 @@ In future, this will be generated from kubernetes/enhancements kep.yaml files
 -->
 
    - Stable
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md) - $milestone.stable
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md) - $milestone.stable
    - Beta
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md) - $milestone.beta
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md) - $milestone.beta
+     - [2436 - Leader Migration for Controller Managers](https://github.com/kubernetes/enhancements/tree/master/keps/sig-cloud-provider/2436-controller-manager-leader-migration) - 1.22
    - Alpha
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md) - $milestone.alpha
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md) - $milestone.alpha
+     - [2133 - Kubelet Credential Provider](https://git.k8s.io/community/$link/README.md) - 1.20
    - Pre-alpha
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md)
+     - [2699 - KEP for adding webhook hosting capability to the CCM framework](https://github.com/cheftako/enhancements/blob/master/keps/sig-cloud-provider/2699-add-webhook-hosting-to-ccm/README.md)
 
 ## Project health
 
 1. What areas and/or subprojects does your group need the most help with?
    Any areas with 2 or fewer OWNERs? (link to more details)
 
    -
-   -
-   -
 
-2. What metrics/community health stats does your group care about and/or measure?
+2. What metrics/community health stats does your group care about and/or
+   measure?
 
-   -
-   -
-   -
+   - We don't currently measure any statistics regarding community health.
 
-3. Does your [CONTRIBUTING.md] help **new** contributors engage with your group specifically by pointing
-   to activities or programs that provide useful context or allow easy participation?
+3. Does your [CONTRIBUTING.md] help **new** contributors engage with your group
+specifically by pointing to activities or programs that provide useful context
+or allow easy participation?
 
-   -
+   - Adding a new CONTRIBUTING.md in the same PR as this report.
```

Comment:
Would suggest noting as a follow-up to reference this doc in your subproject repos.